ui/src/shared/components/cells/Cell.scss

```diff
@@ -29,6 +29,10 @@ $cell--header-size: 36px;
   .time-machine-tables {
     padding-top: 0;
   }
+
+  .vis-plot-container {
+    padding-top: 5px;
```

Comment:
Small nit, but would be nice to see use of the global margin variables, which would shoe horn this into either `4px ` or `6px`

`$ix-marg-a`
or
`$ix-marg-a + $ix-border`

Reply:
Ah sorry, this is like the millionth time you've told me this. Will update!